jni/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheImpl.java

```diff
@@ -935,12 +942,142 @@ public ServerStatistics stats() {
             buf[i] = (byte) executeResult.get(i);
         }
         try {
+        	if (buf.length != 0 ) {
             return (T) jb.objectFromByteBuffer(buf);
+        	}
```

Comment:
This should be `} else {` and the indentation looks a little weird here too. The `if (buf.length != 0 )` statement above doesn't line up with the `return`.

Reply:
ok done